### PR TITLE
Implement includes?

### LINKED ISSUE #53: Implement includes? function

### PR DESCRIPTION
**Pull request by VoxSecundus**
Fixes #53

### CODE REVIEW

File: lib/rudash/includes.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module Rudash
+  module Includes
+    def includes?(array, value)
+      array.include?(value)
```
Comment (Attrash-Islam) [inline lib/rudash/includes.rb]: This implementation is a little problematic since it's not aligned with Lodash. One of the purposes of Rudash is to supply the same Lodash API for JavaScript developers going into the Ruby world with their own favourite utility.

In your includes? the output would be:
```ruby
[{a: 1}].includes?({a: 1}) # => true
```

In Lodash it would be:
```js
_.includes([{a: 1}], {a: 1}) // false
```

.. And that's because _.includes checks for reference equality and not the content values, you can fix this by using a combination of the already implemented Rudash functions: `eq?` (Checks for reference equality), `every?` (Iterates on an array with a predicate function).

_**Reminder**_: You can access the already implemented Rudash methods by using **`self`** without needing to require anything in this file 🙂

File: test/includes.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+require_relative '../lib/rudash'
+require 'test/unit'
+
+class IncludesTest < Test::Unit::TestCase
```
Comment (Attrash-Islam) [inline test/includes.rb]: Can you please add a unit tests for `nil`, `hash`?

Comment (Attrash-Islam) [reply test/includes.rb]: See **`Examples`** section here to add more unit tests -> https://lodash.com/docs/4.17.15#includes

**Comment by VoxSecundus** [comment]
Hi, thanks for the comments! Apologies for the implementation errors, clearly didn't do my research into Lodash enough. I'll be taking a look tomorrow. Appreciate the welcoming to this project :blush:

**Comment by Attrash-Islam** [comment]
If you need any help ping me. I advice you to have a look on the methods that support Collections in Rudash such as _map_ -> https://github.com/Attrash-Islam/rudash/blob/master/lib/rudash/map.rb

**Comment by Attrash-Islam** [comment]
Hi @VoxSecundus, I'm sorry but I need to close this PR since the structure of the things has changed a little bit 🙁 Resend a new one if you're still working on it. Thanks 🌹
